cmd/config.go

```diff
@@ -0,0 +1,75 @@
+package cmd
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+
+	guid "github.com/google/uuid"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+// configCmd represents the config command
+var configCmd = &cobra.Command{
+	Use:   "config",
+	Short: "Links uuid with Snyk token.",
+	Long: `config registers Snyk Token 
+	with UUID on crda server and outputs UUID.`,
+	Args: validateArgs,
+	Run:  main,
+}
+
+func init() {
+	rootCmd.AddCommand(configCmd)
+
+}
+func validateArgs(cmd *cobra.Command, args []string) error {
+	if len(args) < 1 {
+		return errors.New("snyk token is missing")
+	}
+	return nil
+}
+
+func requestServer(synkToken string) string {
```

Comment:
There should be a highlevel package to represent login/auth. Don't bind everything inside cmd. cmd can have functionalities related to cli, rest should have individual high level packages with clean interface to interact with those.

Reply:
> Don't bind everything inside cmd 
Its not true. `config.go` is specific to crda config command only.